resources/content/config-content/cluster/content/etc/init.d/cluster

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+### BEGIN INIT INFO
+# Provides:		cluster
+# Default-Start:	2 3 4 5
+# Default-Stop:		0 1 6
+# Short-Description:	swarm cluster
+# Description:		swarm cluster
+### END INIT INFO
+
+PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
+SWARM_ARG_FILE=/etc/swarm_args.conf
```

Comment:
Not sure.  There's a bunch of other config files currently in the base of /etc
I was tempted to maybe create a /etc/cluster directory and put it in there